R/zoop_synthesizer.R

```diff
@@ -573,7 +574,8 @@ Zoopsynther<-function(
   }
 
   Zoop<-Zoop%>%
-    dplyr::mutate(Source = dplyr::recode(.data$Source, twentymm = "20mm"))
+    dplyr::mutate(Source = dplyr::recode(.data$Source, twentymm = "20mm")) %>%
+    dplyr::distinct()
```

Comment:
Why was this change made?

Reply:
I was trying to figure out why some things got duplicated, but I think I can remove this now. I'll change it back.